### ISSUE TITLE
About the training intermediate result.

### ISSUE THREAD

**Issue by daiyixiang666** [valeoai/Maskgit-pytorch#13]
Hello, really thanks for your great work, 
I have a question about the intermediate result. I am trying to reproduce the result in the video domain, but it is really hard to train and the loss does not drop significantly and it keeps producing the pure image where this whole image contains only one color. I just want to ask is it also true for the intermedia result in the Maskgit ? 
Really looking forward and thanks for your reply

**Comment by llvictorll** [comment]
Hello, 

I did not have this problem when training on ImageNet. The images looked bad in the beginning, but in a constant improvement all along the training, as well as the loss. After ~50epochs the object in the generated image was quite recognizable. 

Best, 

Victor
